app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h2>List authors</h2>
\ No newline at end of file
+<div>
+<% @users.each do |user| %>
+<div class="user-container">
+      <div class="photo"></div>
+      <div class="user-info">
+        <h1><%= link_to user.name, user, :class => 'link'%></h1>
+        <h4>Number of posts: <%= user.posts_counter%></h4>
```

Comment:
- [ optional] instead of adding the for loop like here you can make use of partials 
![image](https://user-images.githubusercontent.com/74744787/216772724-2210285c-618e-4b1b-80c7-e3f1d799eb04.png)